### PR TITLE
Update getting-started-with-github-copilot.md

### PR DESCRIPTION
**Pull request by isaac180296**
### Why:

Closes: 

### Check off the following:

- [ ] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [ ] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

File: content/copilot/getting-started-with-github-copilot.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ topics:
   - Copilot
 ---
 
+hello
```
Comment (isaacmbrown) [inline content/copilot/getting-started-with-github-copilot.md:21]: Well

Comment (isaacmbrown) [inline content/copilot/getting-started-with-github-copilot.md:21]:
```suggestion
hello there
```

File: content/copilot/getting-started-with-github-copilot.md [hunk 2]
```diff
@@ -18,6 +18,8 @@ topics:
...
+
```
Comment (alshskd) [inline content/copilot/getting-started-with-github-copilot.md:22]: https://github.com/github/docs/pull/26374/files#r1247892666

File: content/copilot/getting-started-with-github-copilot.md [hunk 3]
```diff
@@ -18,6 +18,8 @@ topics:
...
 {% data reusables.copilot.copilot-cta-button %}
```
Comment (alshskd) [inline content/copilot/getting-started-with-github-copilot.md:24]: مرحبا هناك

**Review by alshskd — COMMENTED** [review]
مرحبا هناك

**Pull request closed without merging**